test/mixer_test.py

```diff
@@ -534,6 +533,34 @@ def test_get_raw(self):
         finally:
             mixer.quit()
 
+    def test_load_buffer(self):
+        """Test loading from various buffer objects."""
+        mixer.init()
+        try:
+            import array
+            samples = b'\x00\xff' * 24
+            snd = mixer.Sound(samples)
+            raw = snd.get_raw()
+            self.assertTrue(isinstance(raw, bytes_))
+            self.assertEqual(raw, samples)
+            snd = mixer.Sound(bytearray(samples))
+            raw = snd.get_raw()
+            self.assertTrue(isinstance(raw, bytes_))
+            self.assertEqual(raw, samples)
+            arsample = array.array('b')
+            if hasattr(arsample, 'frombytes'):
+                arsample.frombytes(samples)
+            else:
+                arsample.fromstring(samples)
+            snd = mixer.Sound(bytearray(samples))
+            raw = snd.get_raw()
+            self.assertTrue(isinstance(raw, bytes_))
+            self.assertEqual(raw, samples)
+        finally:
+            mixer.quit()
+
+        samples = b'abcdefgh' # keep byte size a multiple of 4
```

Comment:
Could we perhaps split this into separate tests for each buffer type? What does the `samples = b'abcdefgh` at the end do?